third_party/pyth/p2w_autoattest.py

```diff
@@ -137,6 +137,8 @@
 mapping_reload_interval_mins: 1 # Very fast for testing purposes
 min_rpc_interval_ms: 0 # RIP RPC
 max_batch_jobs: 1000 # Where we're going there's no oomkiller
+default_attestation_conditions:
```

Comment:
@drozdziak1 and i agreed that i can update the integration tests in a follow-up, since we currently don't have a good way to validate that the things in the config are actually being attested.

Reply:
:+1: